Request/AbstractRequestBodyParamConverter.php

```diff
@@ -151,21 +155,23 @@ protected function execute(Request $request, ParamConverter $configuration)
     }
 
     /**
-     * @return DeserializationContext
+     * @return DeserializationContext|Context
      */
     protected function getDeserializationContext()
```

Comment:
Ah yes, you're right. Maybe we should backport the serializer API to 1.8, add our own layer for the context and rely on that then.